tools/win32/detect_cpu_extensions_wine.py

```diff
@@ -68,14 +69,12 @@ def path_as_windows(fpath):
 
 def write_summary(allcodes):
     """Write a summary of all found codes to stdout."""
-    print """\n
-----------------------------------------------------------------------------
-Checked all binary files for CPU extension codes. Found the following codes:"""
+    print("\n"
+          "----------------------------------------------------------------------------"
+          "Checked all binary files for CPU extension codes. Found the following codes:")
```

Comment:
I'm not sure if this is ever used. It would only ever work on Python 2 because of the print statement. The same argument applies to `tools/osx/build.py`
The way I started off this PR was to check for "print " in my IDE to see where the remaining print problems were in docstrings. I also ended up changing things like these.